api/routes/apds/activities/goals/put.js

```diff
@@ -74,14 +77,15 @@ module.exports = (
                   description: objective,
                   activity_goal_id: goalID
                 });
-                await objectiveModel.save();
+                alsoAwaiting.push(objectiveModel.save());
```

Comment:
I'm not sure this one is actually necessary...  it might have been fine already...  🤔